plasmapy/physics/distribution.py

```diff
@@ -410,168 +410,6 @@ def Maxwellian_speed_1D(v,
         return distFunc
 
 
-def Maxwellian_speed_3D(vx,
```

Comment:
This can be removed, but `Maxwellian_speed_1D` needs to be renamed to `Maxwellian_speed_3D` and moved here. Docstrings and tests should be changed accordingly. We then need to create a correct implementation for  `Maxwellian_speed_1D`.